common/config/babel/.babelrc.js

```diff
@@ -39,6 +39,8 @@ process.env['COMMUNICATION_REACT_FLAVOR'] === 'stable' &&
         'unsupported-browser',
         // Support Calling SDK isReceiving flag, shows a loading spinner on the video tile when isAvailable is true but isReceiving is false
         'video-stream-is-receiving-flag',
+        // Pinned Participants
+        'pinned-participants'
```

Comment:
should this tag be `videogallery-layouts`? or are we splitting different parts of the feature up?

Reply:
I think it is specific to pinning participants, that is why we opted for pinned-participants. Although we can have any option in the menu, @anjulgarg what do you suggest should we change the conditional compile comment to `videogallery-layouts`?

Reply:
We are splitting different features into smaller parts. This is correct :)